### PR TITLE
Remove unused commands

### PR DESCRIPTION
**Pull request by yaoandrew**
Removes contractWatcher and composeAndExecute

### CODE REVIEW

File: cmd/contractWatcher.go [hunk 1]
```diff
@@ -1,108 +0,0 @@
...
-}
-
-func contractWatcher() {
-	ticker := time.NewTicker(5 * time.Second)
-	defer ticker.Stop()
-
-	blockChain := getBlockChain()
-	db := utils.LoadPostgres(databaseConfig, blockChain.Node())
-
-	con := config.ContractConfig{}
```
Comment (rmulhol) [inline cmd/contractWatcher.go:87]: I think if we're going to delete this command, then we should probably also delete a lot of the downstream code that's only used by it

Comment (yaoandrew) [reply cmd/contractWatcher.go:87]: Oh you mean like really delete it. 😄

**Comment by paytonrules** [comment]
🥳 - and more to delete when you delete the unused code.

**Comment by yaoandrew** [comment]
Closing this for now since deleting the `contractWatcher` command and entity code requires changes the `Exporter` interface here and modifying the `Export() ` in vdb-mcd-transformers repo. Can circle back later.

**Pull request closed without merging**